newrelic-integration-e2e/resources/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+  agent:
+
+    container_name: newrelic-infra
+    image: newrelic/infrastructure:latest
+    cap_add:
+      - SYS_PTRACE
+    network_mode: host
+    pid: host
+    privileged: true
```

Comment:
also here, why do we have two docker-compose?

Reply:
Unused